packages/react-compose/src/types.ts

```diff
@@ -56,7 +56,7 @@ export type ComposeRenderFunction<TElementType extends React.ElementType = 'div'
 export type ComposeOptions<TInputProps = {}, TInputStylesProps = {}, TParentProps = {}, TParentStylesProps = {}> = {
   className?: string;
 
-  classes?: ClassDictionary | ClassFunction;
+  classes?: (ClassDictionary | ClassFunction)[];
```

Comment:
this should be either a single item or an array. most cases you just need one. i don't think it would be much cost to check if array;